host/sgx/loadelf.c

```diff
@@ -720,7 +729,9 @@ static oe_result_t _set_uint64_t_dynamic_symbol_value(
     if (elf64_find_dynamic_symbol_by_name(&image->elf, name, &sym) != 0)
         goto done;
 
-    symbol_address = (uint64_t*)(image->image_base + sym.st_value);
+    OE_CHECK(sym.st_value > image->image_size);
```

Comment:
Why did you add this check?

Reply:
```suggestion
    if (sym.st_value <= image->image_size)
        OE_RAISE(OE_INVALID_PARAMETER);
```
Also, please explain what's the check for.